projects/demo/src/app/modal/modal-trap.demo.html

```diff
@@ -6,14 +6,14 @@
 
 <p>
   For accessiblity purposes, you may want to trap user focus to within the modal. You can use the
-  <code>clrFocusTrap</code> directive to keep the user tabbing from within the modal.
+  <code>cdkTrapFocus</code> directive to keep the user tabbing from within the modal.
 </p>
 
 <div class="clr-example">
   <div class="backdrop-example-container">
     <div class="modal static">
       <div class="modal-dialog" role="dialog" aria-hidden="true">
-        <div clrFocusTrap class="modal-content">
```

Comment:
For some reason, I don't see this working while manually testing. Looks ok in code, but the trap is missing here:
https://431--storybook-clarity-design.netlify.app/demo/modal/focus-trap

Reply:
https://github.com/vmware-clarity/ng-clarity/pull/431#issuecomment-1344410185